public/video-ui/src/pages/Upload/index.jsx

```diff
@@ -84,7 +84,7 @@ class VideoUpload extends React.Component {
                 this.props.uploadActions.s3UploadPostProcessing
               }
               s3UploadReset={this.props.uploadActions.s3UploadReset}
-              activatingAssetNumber={this.props.videoStates.activatingAssetNumber}
+              activatingAssetNumber={this.props.isActivatingAssetNumber.activatingAssetNumber}
```

Comment:
I suspect this should be: 
```suggestion
              activatingAssetNumber={this.props.isActivatingAssetNumber}
```